scripts/docker_cleanup.sh

```diff
@@ -2,7 +2,8 @@
 
 set -xe
 echo "Remove exited containers"
-docker ps -a |awk '/Exited/{print $1}' |while read cid; do docker rm $cid||:; done
+#https://docs.docker.com/engine/reference/commandline/ps/#examples
+(docker ps -a -q -f "status=exited"; docker ps -a -q -f "status=dead")| while read cid; do docker rm $cid||:; done
```

Comment:
Does `docker ps -a -q -f "status=exited" -f "status=dead"` not work?

Reply:
I just tried `(docker ps -a -q -f "status=running" -f "status=created")| while read cid; do echo $cid||:; done`, specifying multiple filters appeared to work fine for me. That said, this is a fairly minor point.